src/menu/main_menu.c

```diff
@@ -184,18 +187,15 @@ static void ignore_p0_handler(engine_ctx_t *ctx,
   ignore_p0_event();
 }
 
-static void get_wallet_id_from_list_selection(uint8_t selection,
-                                              uint8_t *wallet_id) {
-  ASSERT(0 != selection && 4 >= selection);
+static void get_flash_wallet_from_list_selection(uint8_t selection,
+                                                 Flash_Wallet **flash_wallet) {
+  ASSERT(0 != selection && 4 >= selection && NULL != flash_wallet);
```

Comment:
Check the range for selection. I think it can be 4 (1 indexed)

Reply:
No change has been made to the range of selection. The assert checks for 1 to 4 inclusive.